Observers/SalesOrderSaveBeforeObserver.php

```diff
@@ -113,7 +120,8 @@ private function createShipment(OrderInterface $order): void
                 );
 
                 //Trigger exception in order to stop shipment creation and display message to admin
-                throw new \Magento\Framework\Exception\LocalizedException(__($message));
+                //UPD: do not trigger exception one order item problem duting shipment placement not resolved on SK side
+                //throw new \Magento\Framework\Exception\LocalizedException(__($message));
```

Comment:
```suggestion
```

Can we message admin any other way? so we know the order is not synchronized fully?